packages/service-library/src/servicelib/async_utils.py

```diff
@@ -42,14 +42,24 @@ class QueueElement:
 _sequential_jobs_contexts: dict[str, Context] = {}
 
 
-async def cancel_sequential_workers() -> None:
-    """Signals all workers to close thus avoiding errors on shutdown"""
-    for context in _sequential_jobs_contexts.values():
+async def _safe_cancel(context: Context) -> None:
+    try:
         await context.in_queue.put(None)
         if context.task is not None:
             context.task.cancel()
             with suppress(asyncio.CancelledError):
                 await context.task
+    except RuntimeError as e:
+        if "Event loop is closed" in f"{e}":
```

Comment:
Note that you are suppressing a very specific exceptions. Is there another **more reliable** way to filter these exceptions? Where is this coming from?

Some suggestions:
- Is it "normal" that the event loop closes? In general I think this is an anomaly that should be diagnosed and corrected. Therefore, I think this is a warning not an info. 
- If you agree with the latter, I would add more info to debug it e.g. trace and exception